### PR TITLE
Kots cli release only

### PR DESCRIPTION
**Pull request by smithers1221**

### CODE REVIEW

File: cli/cmd/release_update.go [hunk 1]
```diff
@@ -1,11 +1,16 @@
 package cmd
 
 import (
-	"errors"
+	"encoding/json"
+	// "errors"
```
Comment (emosbaugh) [inline cli/cmd/release_update.go]: remove this

File: cli/cmd/root.go [hunk 2]
```diff
@@ -176,6 +180,12 @@ func Execute(rootCmd *cobra.Command, stdin io.Reader, stdout io.Writer, stderr i
 				return err
 			}
 			runCmds.appID = app.ID
+		} else {
```
Comment (emosbaugh) [inline cli/cmd/root.go]:
```
else if appType == "kots" {
```

File: pkg/shipclient/release.go [hunk 3]
```diff
@@ -117,17 +118,29 @@ func (c *GraphQLClient) ListReleases(appID string) ([]types.ReleaseInfo, error)
...
 			return nil, err
 		}
+
+		for _, shipReleaseChannel := range shipRelease.Channels {
+			activeChannel := types.Channel{
+				ID:   shipReleaseChannel.ID,
+				Name: shipReleaseChannel.Name,
+			}
+			activeChannels = append(activeChannels, activeChannel)
+
```
Comment (emosbaugh) [inline pkg/shipclient/release.go]: extra newline

File: client/collector.go [hunk 4]
```diff
@@ -50,6 +45,7 @@ func (c *Client) UpdateCollector(appID string, specID string, yaml string) (inte
 }
 
 func (c *Client) UpdateCollectorName(appID string, specID string, name string) (interface{}, error) {
+
```
Comment (emosbaugh) [inline client/collector.go]: extra newline

File: pkg/kotsclient/release.go [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+
+	releaseInfo := types.ReleaseInfo{
+		AppID:    appID,
+		Sequence: response.Data.KotsReleaseData.Sequence,
+	}
+
+	return &releaseInfo, nil
+}
+
+var updateKotsRelease = `
```
Comment (emosbaugh) [inline pkg/kotsclient/release.go]: is this used?

Comment (smithers1221) [reply pkg/kotsclient/release.go]: oh I could, that had been the intention - I set the variable then just pasted the mutation directly into the UpdateRelease function.  <facepalm>.  will fix to as originally intended.

**Comment by smithers1221** [comment]
yep - got over eager in deleting code to strip down to just the changes pertaining to the 'release' command.  fixed the above, tests passing now.

**Pull request merged**